wordpress-mobile/WordPress-iOS
# Issue/update media picker to 1.4.0

Fixes #https://github.com/wordpress-mobile/MediaPicker-iOS/issues/322

This PR updates the MediaPicker version to sort out the problem mentioned on the ticket above.
Here is a video that shows what was happening before:

![media-loading](https://user-images.githubusercontent.com/651601/56897381-52deb600-6a86-11e9-8fec-28def210236c.gif)

To test:

Open site icon media picker
• Tap into WordPress Media (site has ~1200 media items)
• Scroll down and up the list a lot to initiate loading of a lot of the media items
• Tap back to the Albums list
• Screen should refresh only once when the media library items are downloaded and we have a new poster image for the Media library.


Update release notes:
